migrations/1665502977143-add-display-name-to-user.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { runMigration } = require('../utils/mongodb')
+
+module.exports.up = runMigration(async (db) => {
+  await db.collection('users').updateMany(
+    {},
+    {
+      $set: {
+        displayName: '',
```

Comment:
I left this as just an empty string because the `<PersonalData />` component has the user context, and performs a check to see if there is a `displayName`. If there isn't, it provides the `givenname` and `surname` fields to display instead.